packages/bpk-tokens/src/setOpacity.js

```diff
@@ -0,0 +1,30 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2017 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Color from 'color';
+
+const setOpacity = function (colorToken, opacity) {
```

Comment:
What happens if i pass `colorToken` as `null` ?

Reply:
(or any other unexpected value)

Reply:
An error is thrown by the underlying library

Reply:
`setOpacity(null, 0.5) => "rgba(0, 0, 0, 0.5)"`

Reply:
we should probably prevent that, having that default doesn't look correct to me